src/Domain/CoordinateMaps/KerrHorizon.hpp

```diff
@@ -0,0 +1,68 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+#include "Utilities/DereferenceWrapper.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/TypeTraits/RemoveReferenceWrapper.hpp"
+
+namespace domain::CoordinateMaps {
+
+/*!
+ * Distorts cartesian coordinates \f$X^i\f$ such that a coordinate sphere
+ * \f$\delta_{ij}X^iX^j=C\f$ is mapped to an ellipsoid of constant
+ * Kerr-Schild-radius \f$r\f$, defined as the largest positive root of
+ *
+ *  \f{equation*}{
+ *  r^4 - r^2 (\vec{x}^2 - \vec{a}^2) - (\vec{a}\cdot \vec{x})^2 = 0.
```

Comment:
Could you reference https://arxiv.org/abs/0805.4192 and relate these docs to Eq (47) in that paper? We also have `PointwiseFunctions/AnalyticSolutions/GeneralRelativity/KerrHorizon.hpp`, so please check if the docs are consistent with that, too :)